frontend/src/pages/api/stats/unique-contracts-by-date.csv.js

```diff
@@ -0,0 +1,19 @@
+import StatsApi from "../../../libraries/explorer-wamp/stats";
```

Comment:
I think we don't need these CSV wrappers anymore. I only created them when we didn't have the charts, so we could build the charts in excel